mbq/client/contrib/permissions.py

```diff
@@ -339,6 +362,14 @@ def has_permission(  # noqa: F811
             "has_permission",
             tags={"call": "has_permission", "result": str(result), "scope": scope},
         )
+        self.registrar.emit(
+            "has_permission_completed",
+            person_id,
+            scope,
+            org_ref,
+            ref_type,
+            result=result,
+        )
```

Comment:
We shouldn't do this in the first PR but we could move our metrics stuff into our own hooks if we want to later

Reply:
If you want to do a follow-up then remove the version bump though